.github/workflows/feed-generator-build.yml

```diff
@@ -0,0 +1,78 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: build-feed-generator
```

Comment:
Why do we have these multiple build .yaml files for each app, one build yaml file should be able to build everything.
cc @youngbupark

Reply:
We can merge them to one iteratively after this pr.

Reply:
Sounds good, after moving them to 1 solution, it would be easier to atleast build all with 1 command